apps/ui/src/components/FormSpaceMembers.vue

```diff
@@ -100,54 +100,32 @@ function deleteMember(index: number) {
             </button>
           </template>
           <template #items>
-            <UiDropdownItem v-slot="{ active }">
-              <button
-                type="button"
-                class="flex items-center gap-2 lg:min-w-[200px]"
```

Comment:
On snapshot.org

<img width="373" height="269" alt="Image" src="https://github.com/user-attachments/assets/64408cc0-b877-4b54-a2e2-f58cc2a18c7e" />

With new changes 

<img width="308" height="247" alt="Image" src="https://github.com/user-attachments/assets/c117719e-54b1-4cf3-9b53-09bdd4aa3b69" />

which one is correct?